admin/class-admin-notices.php

```
@@ -270,11 +266,8 @@ public function edac_gaad_notice() {
 		// Check if the current date is within the specified range.
 		if ( $current_date >= EDAC_GAAD_NOTICE_START_DATE && $current_date <= EDAC_GAAD_NOTICE_END_DATE ) {
 
-			// Get the promotional message from a separate function/file.
-			$message = $this->edac_get_gaad_promo_message();
-
-			// Output the message with appropriate sanitization.
-			echo wp_kses_post( $message );
+			// Get and output the promotional message with appropriate sanitization.
+			echo wp_kses_post( $this->edac_get_gaad_promo_message() );
```

PHPStan (2.0.3)

Method EDAC\Admin\Admin_Notices::edac_gaad_notice() should return string but return statement is missing.


(return.missing)